packages/tree/src/TreeItem.ts

```diff
@@ -0,0 +1,31 @@
+export default class TreeItem {
```

Comment:
Why did you opt for a class instead of an object? I'm not saying you should change it. Just trying to learn.

Reply:
I pulled a lot of this from our angular 9 version and it was a class in there, no reason other than that. Typically in our angular Apps we utilized classes, but I am not sure there is a rhyme or reason why. Just a choice